frontend/src/components/data-provider/AddDataSection.tsx

```diff
@@ -0,0 +1,138 @@
+import { Button } from "@components/ui/button";
+
+export default () => {
+  return (
+    <div className="container flex flex-col gap-16 bg-white py-6 text-gray-500 md:flex-row md:py-[96px]">
+      <div className="flex flex-col justify-center md:w-[380px] md:min-w-[380px]">
+        <h1 className="text-4xl font-extrabold text-gray-900">
+          Designed with your data in mind
+        </h1>
+        <p className="my-4 text-xl font-normal text-gray-500">
+          TDC places a strong focus on user-submitted data, empowering
+          individuals and organisations to contribute their valuable data to
+          drive insights and innovations in sustainable transportation.
+        </p>
+        <Button className="bg-[#006064] px-6 py-3.5">Add data</Button>
```

Comment:
Mockup below, as you can see the button is not full width, probably adding a `w-fit` should work 
![image](https://github.com/user-attachments/assets/e86695fb-e9c8-4b79-ad00-00242515ebe7)